iso15118/secc/states/iso15118_2_states.py

```diff
@@ -791,24 +837,57 @@ async def process_message(
             # TODO Either an MO Root certificate or a V2G Root certificate
             #      could be used to verify, need to be flexible with regards
             #      to the PKI that is used.
-            verify_certs(leaf_cert, sub_ca_certs, CertPath.MO_ROOT_DER)
-
-            # TODO Check if EMAID has correct syntax
+            verify_certs(
+                leaf_cert, sub_ca_certs, self._mobility_operator_root_cert_path()
+            )
```

Comment:
Will add a ticket noting that the verification needs to be more dynamic than this.

Reply:
@HugoJP1 Have you done this, or shall I pick it back up?

Reply:
I think this falls under #94 .